stdlib/public/SwiftShims/CoreFoundationShims.h

```diff
@@ -69,6 +69,7 @@ _swift_shims_CFIndex _swift_stdlib_CFStringGetLength(
     _swift_shims_CFStringRef _Nonnull theString);
 
 SWIFT_RUNTIME_STDLIB_INTERFACE
+__attribute__((ns_returns_retained))
```

Comment:
Should _swift_stdlib_CFStringCreateCopy also be annotated? See https://developer.apple.com/library/content/documentation/CoreFoundation/Conceptual/CFMemoryMgmt/Concepts/Ownership.html, "The Create Rule".